tox.ini

```diff
@@ -9,3 +9,4 @@ deps =
     nosexcover
     mock
     coverage
+    unittest2
```

Comment:
No code changed that uses unittest2 - why did we need to add this?

Reply:
@mmphego please see the above comment

Reply:
Tox broke when we were debugging, complaining about missing unittest2

It's being imported by https://github.com/ska-sa/astrokat/blob/579161e0f30a2556ed45d043b4f28fbd18280d14/astrokat/test/test_offline_observe.py#L5 and https://github.com/ska-sa/astrokat/blob/579161e0f30a2556ed45d043b4f28fbd18280d14/astrokat/test/test_nd_timings.py#L5

but never installed, unless one runs `setup.py tests` as defined in https://github.com/ska-sa/astrokat/blob/master/setup.py#L41

Reply:
OK

Reply:
But since, we only needed it when we were debugging tests, do you think it's necessary to add it in tox.ini?

Also, I do not think we are being efficient. Why am I saying this, we are adding test dependencies both under `tox.ini` as well as `setup.py`.

Reply:
@amakhaba or @bngcebetsha perhaps remove test dependencies in https://github.com/ska-sa/astrokat/blob/master/setup.py#L41 and append them under `tox.ini`.

You might also consider updating `tox.ini`
see: https://github.com/ska-sa/katlogger/blob/master/tox.ini